packages/terra-theme-provider/tests/jest/ThemeProvider.test.jsx

```diff
@@ -14,9 +18,42 @@ describe('ThemeProvider', () => {
     expect(wrapper).toMatchSnapshot();
   });
 
-  it('should render a theme provider component', () => {
-    const wrapper = render(defaultRender);
-    expect(wrapper).toMatchSnapshot();
+  it('verify theme name is added to documentElement class list', () => {
```

Comment:
nit: can we word these so they sound "sentence-like", i.e. "it verifies theme name is added..."?

Reply:
I will make this change.